app/src/main/java/org/simple/clinic/summary/compose/StatinNudgeView.kt

```diff
@@ -0,0 +1,151 @@
+import androidx.compose.material3.CardDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.simple.clinic.R
+import org.simple.clinic.common.ui.theme.SimpleTheme
+
+@Composable
+fun StatinNudge(
+    isVisible: Boolean,
+    modifier: Modifier = Modifier
+) {
+  SimpleTheme {
+    AnimatedVisibility(
+        visible = isVisible,
+    ) {
+      Card(
+          modifier = modifier
+              .fillMaxWidth(),
+          shape = RoundedCornerShape(4.dp),
+          elevation = CardDefaults.cardElevation(4.dp)
+      ) {
+        Column(
+            modifier = Modifier
+                .background(SimpleTheme.colors.onToolbarPrimary)
+                .padding(16.dp),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+          Row {
+            Box(
+                modifier = Modifier
+                    .weight(2f)
+            )
+            Column(
+                modifier = Modifier
+                    .weight(3f),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+              Text(
+                  modifier = Modifier
+                      .background(SimpleTheme.colors.error, shape = RoundedCornerShape(50))
+                      .padding(horizontal = 8.dp, vertical = 4.dp),
+                  style = SimpleTheme.typography.button,
+                  color = SimpleTheme.colors.onToolbarPrimary,
+                  text = stringResource(R.string.statin_alert_at_risk_patient)
+              )
+            }
+          }
+          Spacer(modifier = Modifier.height(4.dp))
+          RiskProgressBar()
+          Spacer(modifier = Modifier.height(16.dp))
+          Text(
+              text = buildAnnotatedString {
+                append(stringResource(R.string.statin_alert_refer_to_doctor_for))
+                append(" ")
+                withStyle(style = SimpleTheme.typography.body2Bold.toSpanStyle()) {
+                  append(stringResource(R.string.statin_alert_statin_medicine))
+                }
+              },
+              color = SimpleTheme.colors.error,
+              style = SimpleTheme.typography.body2,
+          )
+        }
+      }
+    }
+  }
+}
+
+@Composable
+fun RiskProgressBar() {
+  Row(
+      modifier = Modifier
+          .fillMaxWidth()
+          .height(16.dp),
+      verticalAlignment = Alignment.CenterVertically
+  ) {
+    Box(
+        modifier = Modifier
+            .weight(1f)
+            .height(4.dp)
+            .background(
+                Color(0xFF00B849),
+                shape = RoundedCornerShape(topStartPercent = 50, bottomStartPercent = 50)
```

Comment:
How about we clip the shape at the parent composable to be `RoundedCornerShape(percent = 50)`, and then iterate over a list of colors to create this segmented color strip? Since these are equally spaced, it should be easier to understand.

Also, there are draw modifiers or we can use `Box` to draw the indicators at a specific position as well if you are concerned about the indicator we need to show for high risk. WDYT?